code/game/objects/structures/stool_bed_chair_nest/alien_nests.dm

```diff
@@ -88,7 +89,10 @@
 	if(user.a_intent != INTENT_HARM)
 		return attack_hand(user)
 	else
```

Comment:
No need for this to be in an else statement, the above code early returns

Reply:
👍